docs/pages/cdp/cdp-Deploying-a-nodejs-application.md

```diff
@@ -53,7 +53,14 @@ CICS TS V5.5 introduced support to run Node.js applications and is required by t
    create : META-INF/cics.xml
    CICS Bundle generated with bundleid "myexpressapp"</pre>
 
-5. Deploy the CICS bundle into CICS.
+5. Confirm that the Node.js SDK is installed on z/OS
+
+   ```text
+   zowe zos-uss issue ssh "node --version" --user <my z/OS userid> --password <my z/OS password>
```

Comment:
The user and password options shouldn't be necessary if they were specified in the profile which they will be if they followed https://ibm.github.io/zowe-cli-cics-deploy-plugin/cdp-Creating-Zowe-CLI-profiles.html#zosmf-profile